doc/how_to/combine_recordings.rst

```diff
@@ -0,0 +1,82 @@
+How to Combine Recordings in SpikeInterface
+===========================================
+
+In this tutorial we will walk through combining multiple recording objects. This can happen when the probe is not
+moved within the tissue, but an overall recording session is split into several subsessions (e.g. pre-stim, stim, post-stim).
+Thus one may wish to combine these individual recording objects into one recording object before sorting.
+
+**Why Combine?**
+
+Spike sorters seek to sort spikes within a recording into groups of units. Thus if multiple :code:`Recording` objects have the
```

Comment:
Nice section. It was useful for me. I will add a concise answer at the beginning and then elaborate in the way that you do.

Something like:
Why combine?
Combining your data into a single recording allows to have consistent labels (.e.g unit_ids) across the whole recording.

What do you think?

Reply:
Sounds good. Will do.